website/docs/configuration/functions/parseint.html.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: "functions"
+page_title: "parseint - Functions - Configuration Language"
+sidebar_current: "docs-funcs-numeric-parseint"
```

Comment:
It's import also to link to this page from the functions navigation bar. You can find the template for that in `website/layouts/functions.erb`.

Reply:
Do I directly make changes to `website/layout/functions.erb`? Or something else?